tests/login.spec.js

```diff
@@ -0,0 +1,28 @@
+const { test, expect } = require('@playwright/test');
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+});
+
+test('login', async ({ page }) => {
+    await page.goto('https://www.saucedemo.com/');
```

Comment:
1. You don't need to go to the main page twice. Leave one. OR in the before each OR inside the test
2. Please, move baseURL to the playwright config instead of harcoding it inside tests. 
- In the config u'll have baseURL: "https://www.saucedemo.com" (property baseURL commented somewhere in the config file, you need to find it, uncomment and change it's value)
- In the tests u'll open the main page like ``` await page.goto('/')```